### PR TITLE
Include postgres port in prod copy rake task

### PR DESCRIPTION
**Pull request by rbarreca**
A quick hack to get the port in there.

### CODE REVIEW

File: lib/tasks/production-copy.rake [hunk 1]
```diff
@@ -205,6 +205,11 @@ module CloneProductionHelper
     configs["development"]["database"]
   end
 
+  def local_development_db_port
+    configs = YAML.load_file(File.new("config/database.yml"))
+    configs["development"]["port"]
```
Comment (thermistor) [inline lib/tasks/production-copy.rake]: Why do you need the port? Are you running on a non-standard port? Why?

Comment (rbarreca) [reply lib/tasks/production-copy.rake]: To run OFN and LO locally they have different postgres requirements so I'm running multiple versions that each run on their own port. https://medium.com/keeping-code/running-multiple-postgresql-versions-simultaneously-on-macos-linux-90b3d7e08ffd

Comment (thermistor) [reply lib/tasks/production-copy.rake]: Let's discuss in our next call.

Comment (rbarreca) [reply lib/tasks/production-copy.rake]: Would be good to get this into master or refactored to be nicer. I keep having to copy this fix to my feature branches.

Comment (thermistor) [reply lib/tasks/production-copy.rake]: Do you time today for a quick chat?

**Comment by rbarreca** [comment]
Closing and not gonna deal with multiple postgres versions.

**Pull request closed without merging**